source/performance/F0.Analyzers.Benchmarks/F0.Analyzers.Benchmarks.csproj

```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="BenchmarkDotNet" Version="0.13.1" />
+    <PackageReference Include="BenchmarkDotNet" Version="0.13.5" />
```

Comment:
Current version is `0.13.8`.

Reply:
Good catch!
Updating to `0.13.6` or greater currently causes _NuGet Error NU1107_:
```
Version conflict detected for Microsoft.CodeAnalysis.Common. Install/reference Microsoft.CodeAnalysis.Common 4.1.0 directly to project F0.Analyzers.Benchmarks to resolve this issue.
  F0.Analyzers.Benchmarks -> BenchmarkDotNet 0.13.8 -> Microsoft.CodeAnalysis.CSharp 4.1.0 -> Microsoft.CodeAnalysis.Common (= 4.1.0)
  F0.Analyzers.Benchmarks -> Microsoft.CodeAnalysis.CSharp.Workspaces 4.0.1 -> Microsoft.CodeAnalysis.Common (= 4.0.1)
```

At first, I didn't want to update any Roslyn packages. But now I guess it should be fine if we update all Roslyn packages, except from the actual Analyzer project `F0.Analyzers.csproj`, as this would be a breaking change. Actually, we're updating our toolchain to the _.NET 7 SDK_ in #106, so we run our tests potentially against the latest SDK - or at least _SDK 7.0.100_ - anyway.
Also, while taking a look at the [API Breaking Changes](https://github.com/dotnet/roslyn/blob/main/docs/Breaking%20API%20Changes.md), keeping the older Roslyn API surface area won't guard us against potentially breaking changes.

Perhaps, to achieve what I originally intended, we have to update our Unit-Test CI workflow to run against multiple .NET SDKs.

Anyway ... I'm now thinking to update _all_ Roslyn packages, _except_ the "core" _.NET analyzers_ project,
- to `4.4.0`, compatible with _SDK 7.0.100_
- to `latest`, currently `4.7.0`, compatible with _SDK 7.0.400_

I prefer the former over the latter, as this resembles the `global.json` from #106:
```
{
  "sdk": {
    "version": "7.0.100",
    "rollForward": "feature"
  }
}
```
and our project is runnable/testable from any _.NET 7 SDK_, while still retaining compatibility with _SDK 6.0.100_ for our resulting NuGet package.

What do you think? Which option do you prefer?